### PR TITLE
improve kubernettes application pages

### PR DESCRIPTION
**Pull request by andreipope**
improve kubernettes application pages

### CODE REVIEW

**Comment by harsh-px** [comment]
@andreipope can you attach screenshots on how the resulting pages look?

In general, let's do that for most PRs that affect look and feel.

File: content/portworx-install-with-kubernetes/application-install-with-kubernetes/cassandra/_index.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-title: Cassandra on Kubernetes
-linkTitle: Cassandra
+title: Cassandra on Kubernetes on Portworx
+linkTitle: Cassandra on Portworx
```
Comment (harsh-px) [inline content/portworx-install-with-kubernetes/application-install-with-kubernetes/cassandra/_index.md]: make all linkTitles just "Cassandra". No need of "on Portworx".

linkTitles show up on sidebar. So the word "on Portworx" is redundant there


So this for all other pages in the PR.

Comment (andreipope) [reply content/portworx-install-with-kubernetes/application-install-with-kubernetes/cassandra/_index.md]: @harsh-px done

**Comment by andreipope** [comment]
> @andreipope can you attach screenshots on how the resulting pages look?
> 
> In general, let's do that for most PRs that affect look and feel.



> @andreipope can you attach screenshots on how the resulting pages look?
> 
> In general, let's do that for most PRs that affect look and feel.

homepage
cassandra- discussion forum
cassandra-measuring-performance-discussion-forum
cassandra-snapshots-discussion-forum
elastic-search-and-kibana-discussion-forum
postgresql-discussion-forum
kafka-with-zookeeper-discussion-forum
mysql-failover-discussion-forum
wordpress-discussion-forum

@harsh-px Sure. I've attached the screenshot. There's an annotation left on the wordpress screenshot. Please let me know your thoughts.

**Pull request merged**